app/routes/items/detail.js

```diff
@@ -38,6 +38,8 @@ export default AuthorizeRoute.extend({
         }
       );
     }
+    this.store.findAll("restriction", { reload: true });
```

Comment:
you need to await the promises. As a general rule never have non-awaited promises unless it's intended